src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -53,9 +53,7 @@ public static Logger getLogger(String name) {
      * Creates a Logger for the given class name.
      */
     public static <T> Logger getLogger(Class<T> clazz) {
-        if (clazz == null) {
-            return Logger.getLogger("");
-        }
+        assert clazz != null;
```

Comment:
I remember we use a different method (rather that `assert`) to check for null parameters elsewhere in AB3. If that is the case, we can do the same, for consistency.